randovania/games/samus_returns/logic_database/Area 5 Tower Interior.txt

```diff
@@ -474,6 +449,38 @@ Extra - asset_id: collision_camera_007
                       Movement (Advanced)
   > Event - Grapple Block
       Grapple Beam
+  > Below Pickup
+      Screw Attack
+
+> Below Pickup; Heals? False
+  * Layers: default
+  > Door to Plasma Beam Chamber
```

Comment:
going up from this door to the interior teleporter door isnt necessarily possible to do if you never shuffled the shuffler.

Reply:
Well this is pickup to lower door so that's not relevant here. Bottom door to top door might not be possible yes.

Reply:
yes, i was referring lower door to top door. i just couldnt comment on any other line